app/controllers/foreman_deployments/api/v2/stacks_controller.rb

```diff
@@ -17,6 +17,11 @@ def create
           ForemanDeployments::StackParser.parse(@stack.definition)
           process_response @stack.save
         end
+
+        api :GET, '/stacks/', N_('List saved stacks')
+        def index
+          @stacks = Stack.all
```

Comment:
If wwe are using the framework, it should reference the `resource_scope` method:
``` ruby
@stacks = resource_scope
```